### PR TITLE
Checkpoint 2 User Sign Up complete

### PR DESCRIPTION
**Pull request by huskergirl6891**
Finished the user story.  Still need to figure out TDD with Devise but wanted to try out my first pull request.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,13 +2,26 @@
...
+    <ul class="nav nav-tabs">
+      <li><%= link_to "Blocipedia", welcome_index_path %></li>
+    </ul>
+    <% if user_signed_in? %>
+      Signed in as: <strong><%= current_user.email %></strong> |
+      <%= link_to 'Edit profile', edit_user_registration_path, :class => 'navbar-link' %> - <%= link_to "Sign Out", destroy_user_session_path, method: :delete, :class => 'navbar-link'  %>
+    <% else %>
+      <%= link_to "Sign Up", new_user_registration_path, :class => 'navbar-link'  %> -
+      <%= link_to "Sign In", new_user_session_path, :class => 'navbar-link'  %>
+    <% end %>
```
Comment (christine-le) [inline app/views/layouts/application.html.erb:21]: Have you thought about adding error handling yet to fulfill these two scenarios:  What happens if you attempt to sign up with an invalid email?  What happens if you attempt to sign up with a duplicate email?

You can simply use the flash messages here to display validation errors to the user (Similar to how you did in bloccit).

File: config/environments/production.rb [hunk 2]
```diff
@@ -76,4 +76,5 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+  config.action_mailer.default_url_options = { host: 'afternoon-fortress-49236.herokuapp.com' }
```
Comment (christine-le) [inline config/environments/production.rb:79]: awesome!  were you able to get everything deployed successfully to Heroku?  i think had trouble accessing the sign up/login page.   so am assuming it must not have the latest code deployed yet.

**Review by christine-le — APPROVED** [review]
Disregard my previous comment!  Your validations work as expected for invalid _sign up_ scenarios.  Thanks!

File: config/environments/production.rb (hunk 2 above)
Comment (huskergirl6891) [reply config/environments/production.rb:79]: Yes, it is deployed to Heroku with the updates from this checkpoint and appears to be working.

**Pull request merged**